template/en/default/pages/quicksearch.html.tmpl

```diff
@@ -389,10 +389,10 @@
   <a href="[% basepath FILTER none %]query.cgi?format=advanced">Advanced Search</a> form.</p>
 
 <ul class="qs_help">
-  <li>All [% terms.bugs %] reported by userA@company.com or assigned to him
+  <li>All [% terms.bugs %] reported by users@company.com or assigned to him
```

Comment:
This is intentional as it means User A (userA@company.com).

Reply:
```suggestion
  <li>All [% terms.bugs %] reported by userA@company.com or assigned to him
```